Packages/vcs/Lib/Canvas.py

```diff
@@ -865,7 +865,7 @@ def __init__(self, gui=0, mode=1, pause_time=0,
         self._canvas_id = vcs.next_canvas_id
         self.ParameterChanged = SIGNAL('ParameterChanged')
         vcs.next_canvas_id += 1
-        self.colormap = "default"
+        self.colormap = None
```

Comment:
Because there are three spots where cmap can be set, so I need to know if user did set it or not.